Podfile.lock

```diff
@@ -1,5 +1,8 @@
 PODS:
   - Alamofire (4.5.1)
+  - Crashlytics (3.9.3):
+    - Fabric (~> 1.7.2)
```

Comment:
do we need this second Fabric (~> 1.7.2)? Does Crashlytics not work on Fabric 1.7.3?

Reply:
nevermind